src/main/scala/chisel3/simulator/Simulator.scala

```diff
@@ -56,7 +56,6 @@ trait Simulator[T <: Backend] {
   def workingDirectoryPrefix = "workdir"
   def customSimulationWorkingDirectory:   Option[String] = None
   def verbose:                            Boolean = false
-  def firtoolArgs:                        Seq[String] = Seq()
```

Comment:
I'm not worried about this breakage because it would require someone implementing their own `Simulator` which is an insanely large task. I searched Github and, shocker, no one has in open-source at least.